sdk/eventgrid/azure-messaging-eventgrid/CHANGELOG.md

```diff
@@ -1,7 +1,21 @@
 # Release History
 
-## 2.0.0-beta.4 (Unreleased)
-
+## 2.0.0-beta.4 (2020-02-10)
+### Breaking changes
+- `CloudEvent` constructor now accepts parameter "data". Removed `setData()`.
+- `CloudEvent.parse()` and `EventGridEvent.parse()` are renamed to `fromString()`.
+- `CloudEvent::getData()` of CloudEvent and EventGridEvent now returns `com.azure.core.util.BinaryData`. 
+  Users can use methods `BinaryData` to deserialize event data. The generic version of `getData()` is then removed.
+- Removed `CloudEvent::getDataAsync()`
+- Added `EventGridSasGenerator` class and removed `EventGridSasCredential`. Use `EventGridSasGenerator.generateSas()` to
+  create a Shared Access Signature and use `AzureSasCredential` to build a `EventGridPublisherClient`.
+- Renamed `sendEvents` to `sendEventGridEvents`
+
+### Dependency Updates
+- Update `azure-core` dependency to `1.13.0`.
+- Update `azure-core-http-netty` dependency to `1.8.0`.
+- Remove dependency on `azure-core-serializer-json-jackson`.
+- Removed dependency on `commons-io`.
```

Comment:
This is not necessary as it was only a test dependency.

Reply:
Removed